machine_learning_experiments/assets/improved-training-set/set.yml

```diff
@@ -6,18 +6,326 @@
+  topics: ["security"]
+- url: "https://docs.bacula-web.org/en/latest/02_install/requirements.html"
+  topics: ["security"]
+- url: "https://docs.celeryq.dev/projects/kombu/en/stable/userguide/serialization.html"
+  topics: ["security", "devops"]
+#- url: "https://docs.godotengine.org/en/stable/classes/class_packetpeer.html"
+#  topics: ["security"]
+
+# Mentions security but not necessarily the best example
+- url: "https://docs.gunicorn.org/en/stable/community.html"
+  topics: ["security"]
+#- url: "https://docs.podman.io/en/latest/markdown/podman-info.1.html"
+#  topics: ["security"]
+- url: "https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/viewconfig.html"
+  topics: ["security"]
+- url: "https://flask-dance.readthedocs.io/en/latest/providers.html"
+  topics: ["security", "devops"]
+- url: "https://oauthenticator.readthedocs.io/en/latest/"
+  topics: ["security", "devops"]
+- url: "https://osu-mac.readthedocs.io/en/latest/issues/malware.html"
+  topics: ["security"]
+- url: "https://snowalert.readthedocs.io/en/latest/index.html"
+  topics: ["security"]
+- url: "https://snowalert.readthedocs.io/en/latest/pages/about.html"
+  topics: ["security"]
+- url: "https://poshc2.readthedocs.io/en/latest/index.html"
+  topics: ["security"]
+
+
+##############################################################################
+# FrontEnd
+##############################################################################
+- url: "https://robkendal.co.uk/blog/2020-02-20-creating-a-react-code-editor-and-syntax-highlighter"
+  topics: ["frontend"]
+- url: "https://robkendal.co.uk/blog/how-to-fix-property-does-not-exist-on-window-type-in-typescript"
+  topics: ["frontend"]
+- url: "http://manual.cytoscape.org/en/stable/Node_and_Edge_Column_Data.html"
+  topics: ["frontend"]
+- url: "https://ae-scripting.docsforadobe.dev/introduction/objectmodel.html"
+  topics: ["frontend"]
+- url: "https://carla.readthedocs.io/projects/ros-bridge/en/latest/ros_compatibility/"
+  topics: ["frontend"]
+- url: "https://doc-kurento.readthedocs.io/en/latest/features/kurento_client.html"
+  topics: ["frontend"]
+
+# HTML standards for open data websites. Seems reasonable.
+- url: "https://docs.ckan.org/en/2.9/contributing/html.html"
+  topics: ["frontend", "datascience"]
+- url: "https://extendscript.docsforadobe.dev/user-interface-tools/index.html"
+  topics: ["frontend"]
+- url: "https://flask.palletsprojects.com/en/1.0.x/tutorial/static/"
+  topics: ["frontend", "backend"]
+- url: "https://git-memo.readthedocs.io/en/latest/subtree.html"
+  topics: ["frontend"]
+- url: "https://immerjs.github.io/immer/"
+  topics: ["frontend"]
+# Mentions JS multiple times
+- url: "https://jupyterlab.readthedocs.io/en/latest/extension/extension_dev.html"
+  topics: ["frontend", "datascience"]
+- url: "https://maxchadwick.xyz/blog/troubleshooting-mismatched-anonymous-define"
+  topics: ["frontend"]
+- url: "https://jsbin.com/"
+  topics: ["frontend"]
```

Comment:
I'm not sure what will TBH. Maybe the content isn't a great fit. I'll check the output.

Reply:
Not amazing but it could be worse.

![Screenshot from 2022-07-08 12-04-27](https://user-images.githubusercontent.com/185043/178054949-e751e506-ea62-4990-8593-3ee4cf6da80a.png)